### PR TITLE
refactor(*): add Warsaw cloud function location (europe-central2)

### PR DESCRIPTION
**Pull request by rosariopfernandes**
This PR should:
- update firebase-admin to `9.8.0`
- update firebase-functions to `3.14.1`
- add Warsaw (`europe-central2`) to the list of available Cloud Function Locations for every extension.

### CODE REVIEW

File: storage-resize-images/extension.yaml [hunk 1]
```diff
@@ -191,14 +193,14 @@ params:
...
-      
-      You may also use wildcard notation for directories in the path. For example, `/users/*/pictures` 
-      would exclude `/users/profile/pictures/image.png` as well as `/users/profile/pictures/any/sub/directory/image.png`. 
-      
-      If you prefer to resize every image uploaded to your Storage bucket, 
+
+      You may also use wildcard notation for directories in the path. For example, `/users/*/pictures`
+      would exclude `/users/profile/pictures/image.png` as well as `/users/profile/pictures/any/sub/directory/image.png`.
+
+      If you prefer to resize every image uploaded to your Storage bucket,
```
Comment (rosariopfernandes) [inline storage-resize-images/extension.yaml:203]: Oops, seems like my IDE got rid of some of the extra whitespaces in those yaml files.

I can revert these changes if needed.

**Review by dackers86 — COMMENTED** [review]
hi @rosariopfernandes.

Thanks for this, looks good to me. We will be looking to merge after the node14 upgrades mentioned last week.

**Comment by dackers86** [comment]
Unable to install and test on PR.

Receiving errors similar to 

```
error TS2430: Interface 'Response' incorrectly extends interface 'ServerResponse'.
```

To avoid build, dependency and doc formatting - moved to https://github.com/firebase/extensions/pull/677

**Comment by jhuleatt** [comment]
Thanks @rosariopfernandes ! We've given you co-author credit on #677

**Comment by rosariopfernandes** [comment]
Thanks for resolving the conflicts and the credits @dackers86 @jhuleatt

**Pull request closed without merging**